packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts

```diff
@@ -119,9 +133,14 @@ const workspaceChecker = new SfdxWorkspaceChecker();
 const parameterGatherer = new EmptyParametersGatherer();
 
 export function createAuthDevHubExecutor(): SfdxCommandletExecutor<{}> {
-  return isDemoMode()
-    ? new ForceAuthDevHubDemoModeExecutor()
-    : new ForceAuthDevHubExecutor();
+  switch (true) {
+    case isSFDXContainerMode():
```

Comment:
I have never seen a switch statement like this...interesting.  Def better then ? : ? : ? :